doc/release/platforms/README.sgi.rst

```diff
@@ -49,8 +55,7 @@ This can be done using the following settings:
 
```

Comment:
Similar to IBM, this file is old, misleading, and wrong. We didn't support qthreads at the time either were written, and I much prefer using the warning as scapegoat for any wrong information that trying to maintain these files over time.